de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/MetadataTemplateViewDialog.java

```diff
@@ -338,28 +465,17 @@ public String getLabel(TemplateAttributeSelectionItem item) {
 
     }
 
-    public IPlantAnchor buildHelpLink(final List<MetadataTemplateAttribute> attributes) {
+    private IPlantAnchor buildHelpLink(final List<MetadataTemplateAttribute> attributes) {
         IPlantAnchor helpLink =
                 new IPlantAnchor(appearance.metadataTermGuide(), 150, new ClickHandler() {
 
                     @Override
                     public void onClick(ClickEvent event) {
-                        VerticalLayoutContainer helpVlc = new VerticalLayoutContainer();
-                        helpVlc.setScrollMode(ScrollSupport.ScrollMode.AUTOY);
-                        for (MetadataTemplateAttribute mta : attributes) {
-                            HTML l = new HTML("<b>" + mta.getName() + ":</b> <br/>");
-                            HTML helpText = new HTML("<p>" + mta.getDescription() + "</p><br/>");
-                            helpVlc.add(l, new VerticalLayoutContainer.VerticalLayoutData(.25, -1));
-                            helpVlc.add(helpText,
-                                        new VerticalLayoutContainer.VerticalLayoutData(.90, -1));
-                        }
-                        Dialog w = new Dialog();
-                        w.setHideOnButtonClick(true);
-                        w.setSize("350", "400");
-                        w.setPredefinedButtons(PredefinedButton.OK);
-                        w.setHeading(MetadataTemplateViewDialog.this.getHeader().getText());
-                        w.setBodyStyle("background: #fff;");
-                        w.setWidget(helpVlc);
+                        MetadataTermGuideDialog w = new MetadataTermGuideDialog(attributes,
```

Comment:
Since we're keeping split points in mind, it might be good to refactor this sort of like [AppDocMarkdownDialog](https://github.com/cyverse-de/ui/commit/4152726f27ed38aa26f5b27c8298ab8421ccff16)